templates/app/assets/javascripts/checkout.js

```diff
@@ -18,17 +18,19 @@ window.addEventListener('DOMContentLoaded', () => {
   const termsCheckbox = document.getElementById('accept_terms_and_conditions');
 
   if (termsCheckbox) {
-    termsCheckbox.addEventListener('change', () => {
-      const submitButton = termsCheckbox.closest('form')
-        .querySelector('[type="submit"]');
-
-      if (termsCheckbox.checked) {
+    const form = termsCheckbox.closest('form')
+    const submitButton = form.querySelector('[type="submit"]');
+    form.onsubmit = function () {
+      if (!termsCheckbox.checked) {
+        termsCheckbox.scrollIntoView(); 
+        alert('Please review and accept the Terms of Service');
         submitButton.removeAttribute('disabled');
-        submitButton.classList.remove('disabled');
+        submitButton.classList.remove('disabled')
```

Comment:
The button is disabled by Solidus.disableSaveOnClick after clicking the "Place Order" button which triggers a form-submit event. This is to prevent a possible double-submit event. I had to clear it in case we do not want to submit the form so the user could click the button again. There does not appear to be any styling associated with disabled such as normally turning grey - I can look into this and make the stylistic changes but I think this also works well as is and maintains previous functionality.

Reply:
Ok, I see. Feel free to ignore the comment in that case.